sway/tree/view.c

```diff
@@ -582,6 +582,14 @@ static struct sway_workspace *select_workspace(struct sway_view *view) {
 	return NULL;
 }
 
+static void update_ext_foreign_toplevel(struct sway_view *view) {
+	struct wlr_ext_foreign_toplevel_handle_v1_state toplevel_state;
+	toplevel_state.app_id = view_get_app_id(view);
+	toplevel_state.title = view_get_title(view);
```

Comment:
Please use initializers when possible: this ensures there are no undefined memory if a new field is added to the struct in a new wlroots version.

```c
const struct wlr_ext_foreign_toplevel_handle_v1_state toplevel_state = {
    .app_id = view_get_app_id(view),
    .title = view_get_title(view),
};
```

Reply:
Done